.cursor/commands/build-and-tests.md

```
@@ -0,0 +1,37 @@
+---
+description: Build the solution and run all tests, fixing any errors iteratively.
+---
+
+## Step 1 — Build
+
+Run the full solution build:
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)